apps/dashboard/app/javascript/packs/files/clipboard.js

```diff
@@ -1,17 +1,41 @@
 import ClipboardJS from 'clipboard'
+import {Handlebars, table} from './datatable.js';
 
+export {ClipboardJS, clipboardjs, clearClipboard, updateClipboardFromSelection, updateViewForClipboard };
 
-window.ClipboardJS = ClipboardJS
+global.ClipboardJS = ClipboardJS
 
-var clipboardjs = new ClipboardJS('#copy-path');
+var clipboardjs = null;

```

Comment:
I don't see a lot of encapsulation here. If you're exporting everything in this file - what's the value add?  Especially when you import the table from the other side.

A theme I'm seeing here is that there is no strong interface between these things.  We may have to take this opportunity to do so - but that seems like a much larger overhaul.